src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -179,23 +179,27 @@ DWORD usbDevicesCount;
 
 // Personalities
 
+#define PERSONALITY_ARABIC              L"ar"
 #define PERSONALITY_BASE                L"base"
+#define PERSONALITY_BENGALI             L"bn"
 #define PERSONALITY_ENGLISH             L"en"
 #define PERSONALITY_SPANISH             L"es"
-#define PERSONALITY_PORTUGHESE          L"pt_BR"
-#define PERSONALITY_ARABIC              L"ar"
+#define PERSONALITY_SPANISH_GT          L"es_GT"
 #define PERSONALITY_FRENCH              L"fr"
-#define PERSONALITY_CHINESE             L"zh_CN"
+#define PERSONALITY_PORTUGUESE_BR       L"pt_BR"
+#define PERSONALITY_CHINESE_CN          L"zh_CN"
 
 static const wchar_t *globalAvailablePersonalities[] =
 {
+    PERSONALITY_ARABIC,
```

Comment:
Apparently I can't drive GitHub, but my point is that the order of this array matters, so we can't sort it into alphabetical order without rearranging all the translations.

https://github.com/endlessm/rufus/blob/9deefca1dc8c9a1e8ac8c156f1ce7e65064eba04/src/endless/res/endless.loc#L232